scripts/src/owners/user_is_repo_owner.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+"""A quick way to check if a given user is an approver in the repository's OWNERS file.
+
+Accepts only a single value (the username)
+
+Returns 0 if the user is found in the OWNERS file in the approver section.
+Returns 1 if the user is NOT found in the OWNERS file.
+Any other non-zero is considered a failed execution (contextually, something broke)
+"""
+
+import os
+import sys
+import yaml
+
+try:
+    from yaml import CLoader as Loader
+except ImportError:
+    from yaml import Loader
+
+OWNERS_FILE = "OWNERS"
+
+
+def is_approver(username: str):
+    """Returns true if username is in the OWNERS file
+
+    Raises an Exception in cases where the content from the OWNERS file
+    does not match our expectations.
+    """
+
+    with open(OWNERS_FILE, "r") as f:
+        data = f.read()
+        out = yaml.load(data, Loader=Loader)
+
+    if "approvers" not in out:
+        raise Exception(
+            'OWNERS file did not have the "approvers" key and it is required'
```

Comment:
This reads as a concatenation of two idea due to the presence of "and". I would simplify to say:
 `OWNERS file does not have the required "approvers" key`
 Alternatively,
 `OWNERS file does not container the "approvers" key which is required`

Reply:
Fixed